.github/ISSUE_TEMPLATE/component_validation_checklist.md

```diff
@@ -0,0 +1,72 @@
+# Component validation checklist
+
+Here is a checklist of items that need to be completed before a component or pattern can be marked as **Ready** in the CMS Design System.
+
+## Documentation is complete
+
+**What to check:**
+- [ ] Description of component or pattern
+- [ ] HTML code snippet example(s)
+- [ ] React code snippet example(s)
+- [ ] All react properties are explained
```

Comment:
Let's capitalize `React` on line 11 and change properties to `props` in code block for clarity of what we're asking be documented.